outsite.nl/components/pages/home/MoreDWH.vue

```diff
@@ -29,7 +29,7 @@ en:
       description: |
         On Mondays we gather around the dinner table to enjoy a delicious meal cooked by one of our
         members, sometimes with an activity afterwards.
-      link: https://dwhdelft.nl/book
+      link: https://dwhdelft.nl/eatingout
```

Comment:
Haal hier de link uberhaupt maar weg, ik denk niet dat er in afzienbare tijd iets nieuws gaat komen, nu is het verwarrend dat je terugspringt naar de homepage. We voegen t wel weer toe als er ooit een nieuw systeem komt